docs/changelog.rst

```diff
@@ -6,6 +6,16 @@ Unreleased
 
 Changelog goes here! Please add your entry to the bottom of one of the lists below!
 
+New features:
+
+* Ability to query albums with track-level (and vice-versa) **db** or
+  **flexible** field queries, for example `beet list -a title:something`, `beet
+  list artpath:cover`.
+* Queries have been made faster, and their speed is constant regardless of
+  their complexity or the type of queried fields. Notably, album queries for
+  the `path` field and those that involve flexible attributes have seen the
+  most significant speedup.
+
```

Comment:
I think that there should be a mention of the interface that changed, along with the fact that it was not public, just to give any plugin developers a hint in case something does go wrong. It's not *really* our responsibility to manage non-public APIs in the program but if it was used in some plugins, it's reasonable to assume it could have been in others.

Reply:
See the last commit - I attempted to keep the same `FieldQuery.field` interface like we had before. What do you think?

Reply:
Ah, then there's no issue at all!